kin-core/src/main/java/kin/core/PaymentWatcher.java

```diff
@@ -19,17 +19,20 @@
 
 /**
  * Watch blockchain network for ongoing payments involved specific account.
- * <p>Use {@link #start(WatcherListener)} to start
- * watching, new payments will be notified on the input {@link WatcherListener WatcherListener&lt;PaymentInfo&gt;}</p>
+ * <p>Use {@link #startPaymentListener(WatcherListener)} to start
+ * watching payment or {@link #startCreateAccountListener(WatcherListener)} to start
+ * watching create account, new payments will be notified on the input {@link WatcherListener WatcherListener&lt;PaymentInfo&gt;}</p>
```

Comment:
I think it will be clearer to separate create account watching from `PaymentWatcher`, and create new `CreateAccountWatcher`, it will not be clear that `PaymentWatcher` is related to create account.

Reply:
will change PaymentWatcher BlockchainWatcher

Reply:
seems good for now